end-to-end-tests-localdb/imageCompare.html

```diff
@@ -0,0 +1,32 @@
+<html>
```

Comment:
@pvannierop bit of duplication here. I'm wondering if there's a better way to re-use this image compare interface with specUtils etc

Reply:
I agree. But what is the best way? What we can do is something like:
```
.\end-to-end-tests
      .\remote
            .\spec
            ...
      .\local
            .\spec
            ...
      .\common
            .\image-compare
            .\lib
            .\wdio
```
What do you think?

Reply:
@pvannierop that seems like a good solution